target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8072-ax880.dts

```diff
@@ -0,0 +1,390 @@
+// SPDX-License-Identifier: MIT
+/* Copyright (c) 2023, Ruslan Isaev <legale.legale@gmail.com> */
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Yuncore AX880";
+	compatible = "yuncore,ax880", "qcom,ipq8074";
+
+	aliases {
+		serial0 = &blsp1_uart5;
+		serial1 = &blsp1_uart3;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		/* Aliases as required by u-boot to patch MAC addresses */
+		ethernet0 = &dp5_syn;
+		ethernet1 = &dp6_syn;
+		label-mac-device = &dp5_syn;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+        bootargs-append = " root=/dev/ubiblock0_1";
```

Comment:
Please use a tab instead of spaces for indentation.

Reply:
Done.

Reply:
Still broken indentation

Reply:
> Still broken indentation

fixed.